andela/odin-ah-frontend
# #162235916 Users should be able to get a list of articles by tag

#### What does this PR do?
- Implements a feature that makes it possible for users to get a list of articles by tag
#### Description of Task to be completed?
- Popular tag links on the landing page should link to a page that displays a list of articles with the selected tag name
- Implement a page that displays articles by tag
#### How should this be manually tested?
- Start the application using npm start
- Go to to localhost:3000
- Click any of the popular tags listed on the right, you should be taken to a page that displays a list of articles with the tag name that was clicked.
- Navigating to `http://localhost:3000/tags/{tag-name}` should also display a list of articles with the "tag-name" specified in the url.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[#162235916](https://www.pivotaltracker.com/story/show/162235916)
#### Screenshots (if appropriate)
<img width="961" alt="screen shot 2018-11-28 at 1 54 42 pm" src="https://user-images.githubusercontent.com/9039613/49153889-52939080-f317-11e8-84f9-cde0e16f1f8e.png">

#### Questions:
